addons/mail/static/tests/chat_window_test.js

```diff
@@ -0,0 +1,46 @@
+odoo.define('mail.chat_window_test', function (require) {
+"use strict";
+
+var testUtils = require('web.test_utils');
+var Widget = require('web.Widget');
+var ChatWindow = require('mail.ExtendedChatWindow');
+
+QUnit.module('mail', {}, function () {
+
+    QUnit.test('chat_window: close using ESCAPE key', function (assert) {
```

Comment:
add a line QUnit.module('chat_window'); above this line.  This is useful to separate properly the tests when looking at them in the logs